etest/__init__.py

```diff
@@ -47,9 +47,17 @@ def echo_check_verbose(check):
 @click.option("-j", "--jobs", default=1, help="number of test to run simultaneously")
 @click.option("-q", "--quiet", is_flag=True, default=False, help="suppress all output")
 @click.option("-v", "--verbose", is_flag=True, default=False, help="provide more output")
+@click.option(
+    "-a",
+    "--arch",
+    "--architecture",
+    default="amd64",
+    type=click.Choice(["amd64", "x86", "arm64", "armv5", "armv7", "ppc64"], case_sensitive=False),
+    help="architecture to test against",
+)
 @click.version_option(information.VERSION)
 @click.argument("ebuilds", nargs=-1)
-def etest(dry_run, fast, jobs, quiet, verbose, ebuilds):
+def etest(dry_run: bool, fast: bool, jobs: int, quiet: bool, verbose: bool, arch: str, ebuilds: str):
```

Comment:
Ebuilds should be a List[str], right?

Reply:
I'm arguing that the type is inconsistent and the fact that mypy isn't complaining is concerning to me.

Reply:
It might be, since tests are failing.

Reply:
The tests unfortunately don't test the typing so I'd doubt it.